form.yml

```diff
@@ -0,0 +1,329 @@
+    $optional: true
+
+  hana_checkbox:
+    $name: Add information about installed HANA database (mandatory for DB/PAS/AAS instances)
+    $type: boolean
+    $default: false
+    $help: Information about the already deployed HANA Database. It is mandatory if DB/PAS/AAS are to be installed
+  hana:
+    $name: Hana database information
+    $visibleIf: .hana_checkbox == true
+    $type: group
+    $help: Information about the already deployed HANA Database. Mandatory for DB/PAS/AAS instances
+    host:
+      $name: Host
+      $type: text
+    sid:
+      $name: SID
+      $type: text
+    instance:
+      $name: Instance
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  schema_checkbox:
+    $name: Add schema information from DB installation
+    $type: boolean
+    $default: false
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+  schema:
+    $name: Schema
+    $visibleIf: .schema_checkbox == true
+    $type: group
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+    name:
+      $name: Schema name
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  product_id:
+    $name: SAP product ID
+    $type: text
+    $help: The installed SAP product id. If some particular node has its own product_id that one will have the preference. The options are. NW750.HDB.ABAPHA, S4HANA1709.CORE.HDB.ABAPHA, S4HANA1809.CORE.HDB.ABAPHA, S4HANA1909.CORE.HDB.ABAPHA. Remove the 'HA' from the end for non HA environment (without ERS)
+    $optional: false
+
+  monitoring_enabled:
+    $name: Enable the host to be monitored by exporters
+    $type: boolean
+    $default: false
+    $help: Enable the node monitoring via exporters which will be installed and configured in all the nodes.
+
+  nodes:
+    $name: Nodes
+    $type: edit-group
+    $minItems: 1
+    $itemName: ""
+    $prototype:
+      host:
+        $name: Hostname to install Netweaver
+        $type: text
+        $optional: false
+      virtual_host:
+        $name: Virtual Hostname
+        $type: text
+        $optional: false
+      ascs_virtual_host_checkbox:
+        $name: Provide the ASCS Virtual Hostname (needed for PAS instance only)
+        $type: boolean
+        $default: false
+        $help: If a nfs share is used to manage the HA mounting point
+      ascs_virtual_host:
+        $name: ASCS Virtual Hostname
+        $visibleIf: .ascs_virtual_host_checkbox == true
+        $type: text
+        $optional: true
+      virtual_host_interface:
+        $name: Virtual host interface
+        $type: text
+        $default: eth0
+        $optional: true
+      virtual_host_mask:
+        $name: Virtual host mask
+        $type: text
+        $default: 32
+        $optional: true
+      sid:
+        $name: SAP system identifier (SID)
+        $type: text
+        $optional: false
+        $help: System ID of the HANA installation
```

Comment:
Maybe add in the `help` that it mast be a 3 characters string and add some examples like `PRD`, `HA1`, etc